app/src/main/java/com/twilio/conversations/app/ui/ConversationDetailsActivity.kt

```diff
@@ -12,18 +12,14 @@ import com.twilio.conversations.app.common.enums.ConversationsError
 import com.twilio.conversations.app.common.extensions.*
 import com.twilio.conversations.app.common.injector
 import com.twilio.conversations.app.databinding.ActivityConversationDetailsBinding
-import kotlinx.android.synthetic.main.activity_conversation_details.*
-import kotlinx.android.synthetic.main.view_add_chat_participant_screen.*
-import kotlinx.android.synthetic.main.view_add_non_chat_participant_screen.*
-import kotlinx.android.synthetic.main.view_conversation_rename_screen.*
 import timber.log.Timber
 
 class ConversationDetailsActivity : BaseActivity() {
-
-    private val renameConversationSheet by lazy { BottomSheetBehavior.from(rename_conversation_sheet) }
-    private val addChatParticipantSheet by lazy { BottomSheetBehavior.from(add_chat_participant_sheet) }
-    private val addNonChatParticipantSheet by lazy { BottomSheetBehavior.from(add_non_chat_participant_sheet) }
-    private val sheetListener by lazy { SheetListener(sheet_background) { hideKeyboard() } }
+    private lateinit var binding: ActivityConversationDetailsBinding
```

Comment:
I see that for ConversationDetailsActivity we have
```
binding = DataBindingUtil
            .setContentView<ActivityConversationDetailsBinding>(this, R.layout.activity_conversation_details)
            .apply {
            lifecycleOwner = this@ConversationDetailsActivity
        }
```
Shall we bring parity to here or in the ConversationDetailsActivity

Reply:
I think it's out of scope for this PR, as this PR only deals with kotlin version update. 
If you think we should align the 2 activities, we can create a separate ticket for that. Lemme know if I should create one.

Reply:
I agree on your point. We can cover it as part of different ticket.